sdk/modelsrepository/Azure.IoT.ModelsRepository/src/ModelsRepositoryMetadata.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace Azure.IoT.ModelsRepository
+{
+    /// <summary>
+    /// ModelsRepositoryMetadata is used to deserialize the contents
+    /// of the Models Repository metadata file as defined by
+    /// https://github.com/Azure/iot-plugandplay-models-tools/wiki/Publishing-Metadata.
+    /// </summary>
+    internal class ModelsRepositoryMetadata
+    {
+        [JsonPropertyName("commitId")]
+        public string CommitId { get; set; }
+        [JsonPropertyName("publishDateUtc")]
+        public string PublishDateUtc { get; set; }
+        [JsonPropertyName("sourceRepo")]
+        public string SourceRepo { get; set; }
+        [JsonPropertyName("totalModelCount")]
+        public int TotalModelCount { get; set; }
+        [JsonPropertyName("features")]
+        public RepositoryFeatures Features { get; set; }
+
+        internal class RepositoryFeatures
+        {
+            [JsonPropertyName("expanded")]
+            public bool Expanded { get; set; }
+            [JsonPropertyName("index")]
```

Comment:
nit: I personally am in favor of new lines between properties.

Reply:
it helps with readability

Reply:
Will make the change